manifest.php

```diff
@@ -1,30 +1,45 @@
 <?php
 
-/*
- * @author CRP Henri Tudor - TAO Team - {@link http://www.tao.lu}
- * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies S.A.
  *
  */
-$extpath = dirname(__FILE__) . DIRECTORY_SEPARATOR;
-$taopath = dirname(dirname(__FILE__)) . DIRECTORY_SEPARATOR . 'tao' . DIRECTORY_SEPARATOR;
+
+$extpath = __DIR__ . DIRECTORY_SEPARATOR;
 
 return [
     'name' => 'taoLtiBasicOutcome',
-  'label' => 'Result storage for LTI',
+    'label' => 'Result storage for LTI',
     'description' => 'Implements the LTI basic outcome engine for LTI Result Server',
-  'license' => 'GPL-2.0',
-  'version' => '3.4.1',
+    'license' => 'GPL-2.0',
+    'version' => '4.0.0',
     'author' => 'Open Assessment Technologies',
     'requires' => [
         'taoResultServer' => '>=4.2.0',
-        'taoLti' => '>=5.0.0'
+        'taoLti' => '>=5.0.0',
```

Comment:
```suggestion
        'taoLti' => '>=11.12.0',
```
due to introduction of `LtiOutcomeXmlFactory`